hw/formal/tools/jaspergold/jaspergold.hjson

```diff
@@ -7,9 +7,10 @@
                "-proj jgproject",
                "-allow_unsupported_OS"]
 
-  // If this line is not seen in the log, then fail the test.
```

Comment:
Thanks Sri, the OneShotCfg does not support default field build_pass_patterns.
If we enable that, we will need to add a build_pass_patterns for all other tools (lint, synthesis, etc).
I think might be easier to add a JG specific failure pattern in the jaspergold.hjson.

Reply:
I think it will be good to add support for this in `OneShotCfg`. It can be left blank for other flows. It will serve as additional proof that the tool ran successfully. WDYT?

Reply:
okay sure, will do that :)